docs/api.rst

```diff
@@ -217,6 +217,47 @@ Helper Functions
     Used for record links to add the appropriate processing annotation to the
     link.
 
+..  class::
+    ConstArray(iterator)
+
+    Used for **Constant Link Values** available since EPICS 3.16.1.
+    Constant Link Values is an EPICS feature which allows passing
+    an list of strings or a list of numbers as a constant into
+    a field which contains a DB link (e.g. INP).
+    See EPICS Release Notes (Section "Constant Link Values")
+    for more details and supported use.
+
+    ConstArray will accept any iterable (e.g. a list) which can generate
+    a non-empty list of values of the same type. Allowed types are:
+
+    * strings and parameters (i.e. :py:class:`epicsdbbuilder.Parameter`)
+
+    * numbers (integers, floating-point, :py:class:`Decimal.Decimal`, and booleans.
+      Booleans will convert to 0 (false) or 1 (true) automatically.
+
+
+    Known limitations:
+
+    * No field type or record type check.
+      ConstArray can be assigned to any field despite the field
+      or the record type does not support Constant Link Values.
+      Use it with link fiels (e.g. INP) of record types stringin,
```

Comment:
"fiels" should be "fields"

Probably best to highlight the record type names (it'll be enough to use `` `...` `` markup for this), ie write:
```
Use it with link fields (e.g. `INP`) of record types `stringin`,
`stringout`, ...
```

Reply:
Typos will be corrected. Markup will be used as suggested.